### PR TITLE
PLT-2490, PLT-2452 Fixing signup flow. Better root redirect. Fixes back buttons.

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: webapp/components/root.jsx [hunk 1]
```diff
@@ -21,10 +24,25 @@ export default class Root extends React.Component {
         };
 
         this.localizationChanged = this.localizationChanged.bind(this);
+        this.redirectIfNessisary = this.redirectIfNessisary.bind(this);
     }
     localizationChanged() {
         this.setState({locale: LocalizationStore.getLocale(), translations: LocalizationStore.getTranslations()});
     }
+    redirectIfNessisary(props) {
```
Comment (jwilander) [inline webapp/components/root.jsx]: `Nessisary` should be spelt `Necessary`

**Comment by jwilander** [comment]
+1, other than spelling mistake

**Comment by crspeller** [comment]
Fixed spelling.

**Comment by hmhealey** [comment]
+1

**Pull request merged**